spec/factories/users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :user do
+    name "Hermione Granger"
+    email "harry@potter.com"
```

Comment:
haha, why can't Hermione have her own email???  props for using HP references though :)